A3A/addons/core/functions/CREATE/fn_createVehicleQRFBehaviour.sqf

```diff
@@ -84,7 +84,7 @@ if (_vehicle isKindOf "Air") then
             if (_vtol != "") then {
                 private _roll = random 100;
 				if(_roll >= 40) then {
-					[_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastrope;
+					[_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastropeVTOL;
```

Comment:
Add a check here to see if vehicle is vtol:
```sqf
if (vehicleIsVtol) then {
    [_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastrope;
} else {
    [_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastropeVTOL;
};
```

Reply:
had the if statement inverted, fixed

Reply:
but yeah, this effectively defeats the whole point of making it a seperate function otherwise

Reply:
this already does that ```
if (_vtol != "")```

> Add a check here to see if vehicle is vtol:
> 
> ```sqf
> if (vehicleIsVtol) then {
>     [_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastropeVTOL;
> } else {
>     [_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastrope;
> };
> ```

Reply:
it's not only does the same thing, it is identical

Reply:
Yes it would appear I was on an older commit